core/App.php

```diff
@@ -399,10 +404,7 @@ public static function handle_request()
                 if ($type == 'create_list') {
                     $clientId = $data['client_id'];
                     $listName = $data['list_name'];
-                    $optIn = $data['opt_in'];

```

Comment:
In creating list, we don't need opt_in type anymore. We pass in false to always create single opt-in.